superx/information_extractors/branch_info_extractor.py

```diff
@@ -88,11 +92,14 @@ def extract_info(self, xml_file):
             elif city is not None:
                 address = address + ' ' + city
 
-            print(branch_id + " " + branch_name + " " + address)
+            sub_chain_id = attrs_dict['sub_chain_id']
+            if type(attrs_dict['sub_chain_id']) is str:
+                sub_chain_id = store.find(attrs_dict['sub_chain_id']).text
+
+            b = Branch(id=branch_id, name=branch_name, address=address, sub_chain_id=sub_chain_id,
+                       chain_id=self.current_super['chain_id'])
+            db.session.add(b)
```

Comment:
What happens if this branch is already in the database?

Reply:
I will check that